spec/features/lettings_log_spec.rb

```diff
@@ -270,6 +270,10 @@
     context "when visiting a subsection check answers page" do
       let(:lettings_log) { create(:lettings_log, :setup_completed) }
 
+      before do
+        lettings_log.owning_organisation.update!(name: "MHCLG")
```

Comment:
Would it be better (fewer database updates) to do `let(:lettings_log) { create(:lettings_log, :setup_completed, owning_organisation: create(:organisation, name: "MHCLG")) }`?

Or even better make the test just use the name from whatever was created rather than comparing to a hardcoded string.